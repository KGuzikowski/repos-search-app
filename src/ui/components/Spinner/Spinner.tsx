import styled, { keyframes } from 'styled-components'
import SpinnerIcon from './spinner.svg'

const rotate = keyframes`
    from {
        transform: rotate(0deg);
    }
    to {
        transform: rotate(360deg);
    }
`

const Spinner = styled.img.attrs({
    src: SpinnerIcon,
  })`
    width: 50px;
    height: 50px;
    margin-top: 50px;
    animation: ${rotate} 1.2s linear infinite;
`

export default Spinner