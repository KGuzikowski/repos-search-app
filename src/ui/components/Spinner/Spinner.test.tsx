import React from 'react'
import { create } from 'react-test-renderer'
import Spinner from './Spinner'

it('renders Spinner component correctly', () => {
    const component = create(<Spinner />)
    expect(component.toJSON()).toMatchSnapshot()
})