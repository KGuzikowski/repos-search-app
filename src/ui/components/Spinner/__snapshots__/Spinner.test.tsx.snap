// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Spinner component correctly 1`] = `
<img
  className="sc-AxjAm BToDG"
  src="spinner.svg"
/>
`;
